### PR TITLE
docs(zh): Polish the Simplified Chinese translations (Part 1)

### PR DESCRIPTION
**Pull request by awxiaoxian2020**
I will see also [translation guide for Vue](https://github.com/vuejs-translations/docs-zh-cn/wiki/翻译须知) to polish the translations.

### CODE REVIEW

File: packages/docs/zh/api/interfaces/pinia.DefineStoreOptions.md
Comment (awxiaoxian2020) [inline packages/docs/zh/api/interfaces/pinia.DefineStoreOptions.md]:
```suggestion
当 store 定义中使用了复杂的 state (如仅客户端的引用)，并且仅从 `pinia.state` 中复制值是不够时，
```

File: packages/docs/zh/api/interfaces/pinia.DefineStoreOptionsInPlugin.md
Comment (awxiaoxian2020) [inline packages/docs/zh/api/interfaces/pinia.DefineStoreOptionsInPlugin.md]:
```suggestion
当 store 定义中使用了复杂的 state (如仅客户端的引用)，并且从 `pinia.state` 中复制值是不够时，
```

File: packages/docs/zh/cookbook/migration-vuex.md
Comment (awxiaoxian2020) [inline packages/docs/zh/cookbook/migration-vuex.md]:
```suggestion
这为 store 创建了一个扁平的结构，但也保留了和之前等价的 `id` 命名方式。如果你在根 store (在 Vuex 的 `store/index.js` 文件中)中有一些 state/getter/action/mutation，你可以创建一个名为 `root` 的 store，来保存它们。
```

**Comment by awxiaoxian2020** [comment]
Because Pinia has a lot of terms to discuss how to translate, this Pull Request will not proofread the wording of the content, only style.

**Comment by awxiaoxian2020** [comment]
@KimYangOfCat 
@ChpShy
@Aimer-Fan
and other contributors,
welcome to review if you have spare time!

**Comment by posva** [comment]
@KimYangOfCat If you can, please, add a good to merge review so I can merge them once ready

**Comment by KimYangOfCat** [comment]
Very willing, but I need a little time to review.

**Comment by posva** [comment]
Of course! Just letting you know that I won’t merge anything until you or others validate the PR 🙂

**Comment by KimYangOfCat** [comment]
my review will be submitted via [the PR](https://github.com/awxiaoxian2020/pinia/pull/1)，so please don't merge this PR before it's merged.

**Comment by awxiaoxian2020** [comment]
@KimYangOfCat I think we can approve these changes now.

**Review by KimYangOfCat — APPROVED** [review]
@posva  LGTM.

**Pull request merged**
